src/osemgrep/language_server/scan_helpers/Scan_helpers.ml

```diff
@@ -143,5 +169,5 @@ let refresh_rules server =
   Lwt.async (fun () ->
       let%lwt () = Session.cache_session server.session in
       end_progress token;
-      scan_workspace server;
+      scan_open_documents server;
```

Comment:
maybe add a comment like
(* we used to call scan_workspace here, but we don't anymore because Bla and Bla *)